core/src/main/java/com/linecorp/armeria/common/logging/DefaultRequestLog.java

```diff
@@ -1274,7 +1273,7 @@ public String toStringResponseOnly(
         }
 
         if (sanitizedContent != null) {                                       // 17
-            buf.append(", content=").append(contentSanitizer.apply(responseContent));
+            buf.append(", content=").append(sanitizedContent);
```

Comment:
Seems like we were sanitizing response content twice by mistake. :sweat_smile:

Reply:
😱